lib/galaxy/model/custom_types.py

```diff
@@ -117,6 +117,8 @@ def compare_values(self, x, y):
 
 
 class DoubleEncodedJsonType(JSONType):
+    cache_ok = True
```

Comment:
Shouldn't this be inherited from `JSONType`?

Reply:
It sure should, but I think there's some extra checks happening in the base class, since you can write custom logic in the subclass that would make this unsafe.

Reply:
https://github.com/sqlalchemy/sqlalchemy/blob/main/lib/sqlalchemy/sql/type_api.py#L1292